File: ProyectoProg3/UtilsArmyModel/src/main/java/pe/edu/pucp/utilsarmy/usuarios/model/Usuario.java

package pe.edu.pucp.utilsarmy.usuarios.model;
import java.util.ArrayList;
import pe.edu.pucp.utilsarmy.gestion_de_prestamos.model.Prestamo;

public class Usuario {
    private int id_usuario;
    private String nombre;
    private String primer_apellido;
    private String segundo_apellido;
    private String DOI;
    private int codigo;
    private String correo;
    
    private String contrasena;
    //private String username;
    private String telefono;
    private int id_rol;
    //private Rol rol_usuario;
    private boolean activa;
    private ArrayList <Prestamo> prestamos;

    public Usuario( String nombre, String primer_apellido, String segundo_apellido, String DOI,
            int codigo,String correo, String contrasena, String telefono,int rol){
        this.nombre = nombre;
        this.primer_apellido = primer_apellido;
        this.segundo_apellido = segundo_apellido;
        this.DOI = DOI;
        this.codigo=codigo;
        this.correo = correo;
        this.contrasena = contrasena;
        //this.username = username;
        this.telefono = telefono;
        this.id_rol=rol;
        
        prestamos=new ArrayList<Prestamo>();
        //this.rol_usuario = rol_usuario;
    }
    public Usuario() {
        prestamos=new ArrayList<Prestamo>();
    }

    public int getId_usuario() {
        return id_usuario;
    }

    public void setId_usuario(int id_usuario) {
        this.id_usuario = id_usuario;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPrimer_apellido() {
        return primer_apellido;
    }

    public void setPrimer_apellido(String primer_apellido) {
        this.primer_apellido = primer_apellido;
    }

    public String getSegundo_apellido() {
        return segundo_apellido;
    }

    public void setSegundo_apellido(String segundo_apellido) {
        this.segundo_apellido = segundo_apellido;
    }

    public String getDOI() {
        return DOI;
    }

    public void setDOI(String DOI) {
        this.DOI = DOI;
    }

    

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }


    public String getContrasena() {
        return contrasena;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public int getId_rol() {
        return id_rol;
    }

    public void setId_rol(int id_rol) {
        this.id_rol = id_rol;
    }

    public boolean isActiva() {
        return activa;
    }

    public void setActiva(boolean activa) {
        this.activa = activa;
    }
    
    @Override
    public String toString() {
        return "id_usuario=" + id_usuario +
                ", nombre='" + nombre + '\'' +
                ", primer_apellido='" + primer_apellido + '\'' +
                ", segundo_apellido='" + segundo_apellido + '\'' +
                ", DOI='" + DOI + '\'' +
                ", codigo=" + codigo +
                ", correo='" + correo + '\'' +
                ", telefono='" + telefono + '\'' +
                ", id_rol=" + id_rol +
                ", activa=" + activa +"\n";
    }

}

File: FrontEnd (C#)/SoftProgModel/GestUsuarios/Usuario.cs
using SoftProgModel.GestPrestamos;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgModel.GestUsuarios
{
    public class Usuario
    {
        private int id_usuario;
        private int codigo_universitario;
        private string nombre;
        private string primer_apellido;
        private string segundo_apellido;
        private int DOI;
        private string contrasena;
        private string correo;
        private string numero_de_telefono;
        private bool activa;
        private Rol rol_usuario;
        private BindingList<Prestamo> prestamos;

        public Usuario()
        {
        }

        public Usuario(int id_usuario, int codigo_universitario, string nombre, string primer_apellido, string segundo_apellido,
            int DOI, string correo, string contrasena, string numero_de_telefono, Rol rol_usuario)
        {
            this.Id_usuario = id_usuario;
            this.Codigo_universitario = codigo_universitario;
            this.Nombre = nombre;
            this.Primer_apellido = primer_apellido;
            this.Segundo_apellido = segundo_apellido;
            this.DOI1 = DOI;
            this.Correo = correo;
            this.Contrasena = contrasena;
            this.Numero_de_telefono = numero_de_telefono;
            this.Rol_usuario = rol_usuario;
        }

        public int Id_usuario { get => id_usuario; set => id_usuario = value; }
        public int Codigo_universitario { get => codigo_universitario; set => codigo_universitario = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Primer_apellido { get => primer_apellido; set => primer_apellido = value; }
        public string Segundo_apellido { get => segundo_apellido; set => segundo_apellido = value; }
        public int DOI1 { get => DOI; set => DOI = value; }
        public string Contrasena { get => contrasena; set => contrasena = value; }
        public string Correo { get => correo; set => correo = value; }
        public string Numero_de_telefono { get => numero_de_telefono; set => numero_de_telefono = value; }
        public bool Activa { get => activa; set => activa = value; }
        public Rol Rol_usuario { get => rol_usuario; set => rol_usuario = value; }
        public BindingList<Prestamo> Prestamos { get => prestamos; set => prestamos = value; }
    }
}
